### ISSUE TITLE
License Zero Manifesto

### ISSUE THREAD

**Issue by dpc** [copyfair/copyfair#10]
Have you seen https://news.ycombinator.com/item?id=15417664 ?

**Comment by fabianhjr** [comment]
Yep, added it to /resources, the interesting one would be L0-reciprocal.

**Comment by kemitchell** [comment]
Drop me a line if you have questions.
